packages/@aws-cdk/aws-rds/test/serverless-cluster.test.ts

```diff
@@ -863,6 +879,158 @@ describe('serverless cluster', () => {
+            },
+            ':SecretString:password::}}',
+          ],
+        ],
+      },
+    });
+
+
+  });
+
+  test('can create a Serverless cluster without vpc but with imported security group', () => {
+    // GIVEN
+    const stack = testStack();
+    const sg = ec2.SecurityGroup.fromSecurityGroupId(stack, 'SG', 'SecurityGroupId12345');
+
+    // WHEN
+    new ServerlessCluster(stack, 'Database', {
+      engine: DatabaseClusterEngine.AURORA_POSTGRESQL,
+      securityGroups: [sg],
+      parameterGroup: ParameterGroup.fromParameterGroupName(stack, 'ParameterGroup', 'default.aurora-postgresql10'),
+    });
+
+    // THEN
+    expect(stack).toHaveResource('AWS::RDS::DBCluster', {
+      Engine: 'aurora-postgresql',
+      DBClusterParameterGroupName: 'default.aurora-postgresql10',
+      EngineMode: 'serverless',
+      MasterUsername: {
+        'Fn::Join': [
+          '',
+          [
+            '{{resolve:secretsmanager:',
+            {
+              Ref: 'DatabaseSecret3B817195',
+            },
+            ':SecretString:username::}}',
+          ],
+        ],
+      },
+      MasterUserPassword: {
+        'Fn::Join': [
+          '',
+          [
+            '{{resolve:secretsmanager:',
+            {
+              Ref: 'DatabaseSecret3B817195',
+            },
+            ':SecretString:password::}}',
+          ],
+        ],
+      },
+      VpcSecurityGroupIds: ['SecurityGroupId12345'],
+    });
+  });
+
+  test('can create a Serverless cluster without vpc but with imported subnet group', () => {
+    // GIVEN
+    const stack = testStack();
+    const subnetGroup = SubnetGroup.fromSubnetGroupName(stack, 'SubnetGroup12345', 'SubnetGroupId12345');
+
+    // WHEN
+    new ServerlessCluster(stack, 'Database', {
+      engine: DatabaseClusterEngine.AURORA_POSTGRESQL,
+      parameterGroup: ParameterGroup.fromParameterGroupName(stack, 'ParameterGroup', 'default.aurora-postgresql10'),
+      subnetGroup,
+    });
+
+    // THEN
+    expect(stack).toHaveResource('AWS::RDS::DBCluster', {
+      Engine: 'aurora-postgresql',
+      DBClusterParameterGroupName: 'default.aurora-postgresql10',
+      EngineMode: 'serverless',
+      DBSubnetGroupName: { Ref: 'SubnetGroup12345' },
+      MasterUsername: {
+        'Fn::Join': [
+          '',
+          [
+            '{{resolve:secretsmanager:',
+            {
+              Ref: 'DatabaseSecret3B817195',
+            },
+            ':SecretString:username::}}',
+          ],
+        ],
+      },
+      MasterUserPassword: {
+        'Fn::Join': [
+          '',
+          [
+            '{{resolve:secretsmanager:',
+            {
+              Ref: 'DatabaseSecret3B817195',
+            },
+            ':SecretString:password::}}',
+          ],
+        ],
+      },
+      VpcSecurityGroupIds: ['SecurityGroupId12345'],
+    });
+  });
+
+  test('can\'t create a Serverless cluster without vpc but with imported vpc subnets', () => {
```

Comment:
```suggestion
  test("can't create a Serverless cluster without VPC but with imported VPC subnets", () => {
```

Please use the same casing in the other test descriptions.

Reply:
idem this could be handled by `prettier` to avoid you loosing your time ;)